src/test/java/br/com/zup/library/utils/TestUtils.java

```diff
@@ -14,4 +18,11 @@
     public String toJson(Object obj) throws JsonProcessingException {
         return this.mapper.writeValueAsString(obj);
     }
+
+    public MockHttpServletRequestBuilder montaRequisicao(Object request,
+                                                         String uri) throws JsonProcessingException {
+        return post(uri)
+                .contentType(APPLICATION_JSON)
+                .content(toJson(request));
+    }
```

Comment:
aqui tenho dúvidas se o nome do método faz jus ao que ele faz. Se amanhã eu for testar uma requisição para `GET`, `PUT` ou `DELETE`, como faz?

Reply:
Posso mudar o nome para montaRequisicaoPost(), ou receber no parâmetro o tipo da requisição. o que me sugere?

Reply:
faz sentido o que vc diz.

mas que tal pensar no idiomisto desse método ao usá-lo com a `MockMvc` ?